app/src/main/res/layout/screen_patient_prescribed_drugs_entry.xml

```diff
@@ -39,4 +39,22 @@
       android:textAppearance="@style/Clinic.V2.TextAppearance.Button1" />
   </org.simple.clinic.widgets.PrimarySolidButtonWithFrame>
 
+  <org.simple.clinic.widgets.PrimarySolidButtonWithFrame
+    android:id="@+id/prescribeddrugs_refill_done"
+    style="@style/Clinic.V2.BottomAlignedButtonFrame"
+    app:buttonBackgroundDisabled="@color/grey2"
+    app:buttonBackgroundEnabled="@color/blue1"
+    app:frameBackgroundDisabled="@color/grey3"
+    app:frameBackgroundEnabled="@color/blue2"
+    android:visibility="gone">
+
+    <com.google.android.material.button.MaterialButton
```

Comment:
The id is in the parent layout for this.

Reply:
Yeah, but we are not supposed to use that. Earlier implementation in this layout is bit wrong, so instead of doing the same thing again let's follow what we do in the rest of the layouts.